src/Features/CSharp/Portable/Formatting/TypingFormattingRule.cs

```diff
@@ -76,7 +71,7 @@ private static bool TryAddSuppressionOnMissingCloseBraceCase(List<SuppressOperat
                 return false;
             }
 
-            if (node.IsKind(SyntaxKind.Block, out BlockSyntax block) && block.Statements.Count >= 1)
+            if (node is BlockSyntax { Statements: { Count: > 1 } statements })
```

Comment:
>= 1, not > 1

Reply:
Reformatted this comment. Nearly didn't understand what you meant because the `>` at the start made it quote.

Reply:
Lol.  Yeah.  That was impossible to understand :D